src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasServer.kt

```diff
@@ -71,16 +73,28 @@ fun main(args: Array<String>) = mainBody {
     val parser = ArgParser(args)
     val arg = CosmasServerArgs(parser)
     LOG.info("Try to bind in port ${arg.port}")
+    val bucket = arg.bucket

```

Comment:
Maybe just remove thias argument? If for whatever reasons we need free and paid buckets to be the same, we'll pass the same argument values. Now with three arguments it looks too complicated both from usage and from implementation perspectives.

Reply:
ok